#! /bin/sh /usr/share/dpatch/dpatch-run
## 16-no-update.dpatch by Michael Meskes <meskes@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Do not check for updates

@DPATCH@
diff -urNad virtualbox-ose-3.0.8-dfsg~/src/VBox/Main/VirtualBoxImpl.cpp virtualbox-ose-3.0.8-dfsg/src/VBox/Main/VirtualBoxImpl.cpp
--- virtualbox-ose-3.0.8-dfsg~/src/VBox/Main/VirtualBoxImpl.cpp	2009-10-07 11:27:14.000000000 +0200
+++ virtualbox-ose-3.0.8-dfsg/src/VBox/Main/VirtualBoxImpl.cpp	2009-10-07 13:44:02.835497353 +0200
@@ -90,6 +90,9 @@
     "<VirtualBox xmlns=\"" VBOX_XML_NAMESPACE "\" "
         "version=\"" VBOX_XML_VERSION_FULL "\">" RTFILE_LINEFEED
     "  <Global>"RTFILE_LINEFEED
+    "    <ExtraData>"RTFILE_LINEFEED
+    "      <ExtraDataItem name=\"GUI/UpdateDate\" value=\"never\"/>"RTFILE_LINEFEED
+    "    </ExtraData>"RTFILE_LINEFEED
     "    <MachineRegistry/>"RTFILE_LINEFEED
     "    <MediaRegistry/>"RTFILE_LINEFEED
     "    <NetserviceRegistry>"RTFILE_LINEFEED
